Lifespark-Technologies/Infomed
# Issue5 backend scaffold

This PR should resolve #5, performs some major refactoring, and creates a backend server scaffold. It creates a Django project in root and puts the frontend in a Django app called `frontend`.

Frontend devs:
* continue developing as usual

Backend devs:
* run `npm run dev` to run webpack and babel to generate `main.js`, which the server will use as the "minified" javascript.
* the HTML file that the server serves is located in `frontend/templates/frontend/index.html` and the `index` function in `frontend/views.py` is what points the server there.
* to create new api and functionality, we can continue to work as though we're working in a regular Django project and create new applications

To test:
Frontend devs:
* to make sure that can still develop normally, you can run `npm start` in the `frontend/` directory. it should start as normal

Backend devs:
* run `python manage.py runserver` to ensure that the server loads the page with the map as expected

To PR reviewers:
* Please perform both frontend and backend tests when reviewing. Thanks a lot.
